src/components/SmallBoard.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="board column is-6">
+    <h2 class="board-name">
+      <slot></slot>
+    </h2>
+    <div
+    v-for="(article, index) in listitems.recent_articles"
+    :key="article.id"
+    class="post">
+      <h3 class="post-title">
+        <router-link
+        :to="{
+            name: 'post',
+            params: { postId: article.id }
+        }">
+        {{index + 1}}. {{ article.title }}
+        </router-link>
+      </h3>
+      <div class="username">
+        {{article.created_by.profile.nickname}}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'the-home-board',
+  props: {
+    listitems: {
+      required: true
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/theme.scss';
+
+.board {
+  max-width: 450px;
```

Comment:
Are there any rules about CSS order?

Reply:
No, unfortunately. Maybe adding linters would be nice.